xmodule/static/README.rst

```diff
@@ -0,0 +1,37 @@
+xmodule/static: edx-platform XBlock assets
+##########################################
+
+Current State: SCSS only
+************************
+
+This folder exists to contain frontend static assets for the XBlocks defined in edx-platform. Its current structure is:
+
+* ``./sass/lms/``: Top-level SCSS modules for student, author, and public views of XModules. Based on LMS styles.
+* ``./sass/cms/``: Top-level SCSS modules for the Studio editor views of XModules. Based on CMS styles.
+* ``./sass/include/``: SCSS modules shared between ``./sass/lms/`` and ``./sass/cms/``.
+* ``./css/``: Git-ignored target for CSS assets, compiled from ``./sass/lms/`` and ``./sass/cms/``.
+
+The top-level XModule styles are added to the Webpack loader at runtime using the custom loader defined in ``xmodule/util/xmodule_django.py``, then included into individual XModules using ``add_webpack_to_fragment``, defined in the same file.
+
+Please note that *XModules* in this context refers to a set of old XBlocks which are defined in the ``xmodule/`` folder of this repository, inherit from ``HTMLSnippet``, and use a special legacy asset processing system. There are a handful of XBlocks in edx-platform that do *not* use this legacy static asset system.
```

Comment:
Maybe explicitly call out the blocks in either category.

Reply:
Done: https://github.com/kdmccormick/edx-platform/tree/kdmccormick/xmodule-static-css/xmodule/static#current-state-scss-only